ehitchhiking-ui/src/app/pages/main-screen/trip-registration/trip-registration.component.sass

```diff
@@ -43,3 +43,7 @@ $trip-registration-form-width: 20rem
   .button-div
     display: flex
     justify-content: center
+  .modal-close-control-btn
+    position: absolute
+    top: -0.4rem
```

Comment:
It is bad practice to use negative values

Reply:
😀